### PR TITLE
fix: Fix incorrect call to client to produce inventory.

### PR DESCRIPTION
**Pull request by kacf**
We use the Provides from the client to generate the inventory, so although it intuitively might look like it doesn't line up, calling `show-provides` from `Inventory` is correct in this instance.

Changelog: Title
Ticket: None

### CODE REVIEW

**Review by lluiscampos — APPROVED** [review]
I'd rephrase the commit reasoning - `show-inventory` command does not exist.

**Comment by kacf** [comment]
> I'd rephrase the commit reasoning - `show-inventory` command does not exist.

Done!
